src/main/play-doc/operation/Install.md

```diff
@@ -536,6 +700,31 @@ A typical response contains the current members of the cluster (shown here is a
 
 ```
 
+### Configuring ConductR Agent
+
+_Repeat each step in this section on each node._
+
+ConductR Agent needs to be connected to a ConductR core node in order for ConductR to run any application process. To establish this connection, configure ConductR Agent as such:
+
+```bash
+[172.17.0.1]$ echo -Dconductr.agent.ip=$(hostname) | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+[172.17.0.1]$ echo --core-node $(hostname):9004 | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+```
+
+By default ConductR Agent and ConductR Core are using `/tmp` as its working directory. Since both ConductR Agent and ConductR Core are installed in the same machine, move the ConductR Agent working directory as such:
+
```

Comment:
@huntc - As discussed, will be handled post-alpha:

https://trello.com/c/k5dbGJJ2/283-ensure-core-and-agent-creates-the-required-bundle-directories